### ISSUE TITLE
Implementation details on restricted prototype access

### ISSUE THREAD

**Issue by papirosko** [peerigon/angular-expressions#37]
As far as i can see, the solution to allow access to properties in parent class (via prototype) is to use eslint. But I can't figure out which rule does the trick. Can you describe your solution in details?

The reason to ask is that i need that access to superclass properties and i have no user input (so security is not a problem)

**Comment by edi9999** [comment]
This "security" has been added here : 

https://github.com/peerigon/angular-expressions/commit/07edb62902b1f6127b3dcc013da61c6316dd0bf1

You could use version 1.0.0 which allowed to access prototype keys.

**Comment by papirosko** [comment]
so security was added inside core angular libraries? i thought they allowed it instead of blocking. Maybe i didn't understand their explanation correctly

**Comment by edi9999** [comment]
No, this was not added to the angular libraries themselves, I added it.

I think the README is not uptodate since it speaks of the fact that there is no security.
